### ISSUE TITLE
Is there a plan to support intel ARC A770? 

### ISSUE THREAD

**Issue by wqwo** [Tugcga/Sycles#25]
Hi, 
   Thanks for your hard working first. Now blender has began to support from V3.6, is there a plan to support intel arc serials video card about sycles 2.x ?

**Comment by Tugcga** [comment]
Yes. In the next update I plan to upgrade the binaries of the Cycles engine and use the latest version from the Blender. But I am not sure when it will happen.
